### PR TITLE
Added Pep 484 Type Hints to the code

### PR DESCRIPTION
**Pull request by SZanko**
Signed-off-by: SZanko <szanko@protonmail.com>

### CODE REVIEW

File: radontea/_alg_art.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import numpy as np
 
 
-def art(sinogram, angles, initial=None, iterations=1,
-        count=None, max_count=None):
+def art(sinogram: np.ndarray, angles: np.ndarray, initial: np.ndarray = None,
+        iterations: int = 1, count=None, max_count=None) -> None:
```
Comment (paulmueller) [inline radontea/_alg_art.py]: Shouldn't `-> None` be `-> np.ndarray`? I should probably add a mypy type check in CI.

File: radontea/_version.py [hunk 2]
```diff
@@ -57,7 +57,7 @@
     import traceback
     import warnings
 
-    def git_describe():
+    def git_describe() -> str:
```
Comment (paulmueller) [inline radontea/_version.py]: Please revert your changes to _version.py, it is a static/stable file shared across repositories and should not be modified at this time. Thanks!

**Comment by paulmueller** [comment]
Hi there, thanks for the PR! Could you please make these two changes and I will be happy to merge!

File: radontea/_alg_art.py (hunk 1 above)
Comment (SZanko) [reply radontea/_alg_art.py]: If you add mypy there would be a problem that mypy doesn't know numpy types without https://github.com/predictive-analytics-lab/data-science-types with is not a complete implementation of the numpy library

Comment (paulmueller) [reply radontea/_alg_art.py]: Ok thanks for that information. Then I won't do it.

**Pull request merged**